ui/packages/keyvis/src/components/KeyVis.less

```diff
@@ -1,9 +1,11 @@
-@import '~antd/es/style/themes/default.less';
+// @import '~antd/es/style/themes/default.less';
+@import '../../../../node_modules/antd/es/style/themes/default.less';
```

Comment:
rollup-plugin-postcss can't recognize '~antd/...` path, should miss some configuration, temporary work around it by using the absolute path.